src/caret_analyze/value_objects/callback.py

```diff
@@ -398,23 +422,27 @@ def __init__(
             Symbol name of the callback.
         subscribe_topic_name : str
             Topic name which the callback subscribes.
-        publish_topic_names : tuple[str, ...] | None
-            Topic name which the callback publishes.
+        subscription_construction_order : int
+            Order of instance creation within the identical subscribes.
+        publish_topics : tuple[PublishTopicInfoValue, ...] | None
+            Topic information which the callback publishes.
         construction_order : int
             Order of instance creation within the identical node.
         callback_name : str | None
             Callback name, by default None. This argument is used by ArchitectureReaderYaml.
 
         """
         self.__subscribe_topic_name = subscribe_topic_name
+        self.__subscription_construction_order = subscription_construction_order
```

Comment:
Can variable `self.__subscribe_topic_name` be replaced by variable `self._subscribe_topic_name`?